### PR TITLE
Update lesson plan page to point to 2021 courses

### PR DESCRIPTION
**Pull request by dmcavoy**
# DONT MERGE UNITL HARD LAUNCH!

Updates the links on code.org/lesson_plans to point to the 2021 courses on Code Studio.

Screen Shot 2021-06-24 at 3 24 01 PM

## Links

https://codedotorg.atlassian.net/browse/PLAT-991

## Testing story

- Manually tested locally

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/course_curriculum.haml
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: if we want to make this future-proof, this could become `https://studio.code.org/s/#{course}`, and as an added bonus you could merge it today and it will get updated automatically on hard launch day. the problem is that the same strategy doesn't work for standards yet. I would say, feel free to keep this PR open as is, and if we happen to implement redirecting for standards then we can update this to lose the `-2021` prefix throughout the PR.

Comment (davidsbailey) [reply pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: https://codedotorg.atlassian.net/browse/PLAT-1125

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: I can update but we had talked about not wanting the redirect for standards because it might be part of the problem of people getting redirected and not knowing and ending up making progress in the wrong year. So it feels like we are going to have to come here to update the standards URL anyway so it might be fine to leave it with the year?

Comment (davidsbailey) [reply pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: ahh, I just realized this won't work for this year, but could be done immediately after hard launch to version-proof for next year.

**Review by davidsbailey — APPROVED** [review]
looks good to merge.

**Comment by davidsbailey** [comment]
(after waiting until hard launch)

**Comment by dmcavoy** [comment]
@davidsbailey I will leave this to you to merge next week

File: pegasus/sites.v3/code.org/views/course_curriculum.haml
Comment (davidsbailey) [reply pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: > I can update but we had talked about not wanting the redirect for standards because it might be part of the problem of people getting redirected and not knowing and ending up making progress in the wrong year. So it feels like we are going to have to come here to update the standards URL anyway so it might be fine to leave it with the year?

what we do not want is urls with version-specific years getting redirected to other version years. redirecting the non-versioned URL to a versioned one should be ok.

regardless, we should hold off on this until after hard launch this year, then future proof for next year.

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/views/course_curriculum.haml:21]: The non-versioned URLs were the concern but I guess not on pages like this. It was a concern that if we put them in resources that hang around year to year like the curriculum guide then you could click a link in the 2019 curriculum guide and all of a sudden be in the 2021 curriculum

**Comment by davidsbailey** [comment]
Closing in favor of https://github.com/code-dot-org/code-dot-org/pull/41401

**Pull request closed without merging**
